backend-api/tests/api-tests/clientCredentialsGrantFlow.test.ts

```diff
@@ -28,11 +33,21 @@ const interceptor = aws4Interceptor({
 axiosInstance.interceptors.request.use(interceptor);
 
 describe("POST /token", () => {
-  describe("Given there is no grant type", () => {
+  let clientIdAndSecret: string;
+
+  beforeAll(async () => {
+    const clientDetails = await getFirstClientDetails();
```

Comment:
Just aligning on terminology with the lambda and services we've built:

The array in Secrets Manager = the Client Registry
An individual item in the array = a registeredClient

Reply:
It's just that we are not getting this information from the client registry - we had to create a new secret and for this reason I went for client details (i.e. to make it clearer that this data isn't coming from the client registry)

Maybe I can call it testClientRegistry instead...